tests/functional/aws-node-sdk/test/multipleBackend/put/put.js

```diff
@@ -198,7 +203,41 @@ describe('MultipleBackend put object', function testSuite() {
                     assert.equal(err, null, 'Expected success, ' +
                         `got error ${err}`);
                     setTimeout(() => {
-                        awsGetCheck(key, correctMD5, correctMD5, () => done());
+                        awsGetCheck(key, correctMD5, correctMD5, awsLocation,
+                          () => done());
+                    }, awsTimeout);
+                });
+            });
+
+            itSkipIfNoEncryption('should encrypt body putting an object to AWS',
```

Comment:
make the assertions in the test conditional based on whether the bucket is encrypted and let's clarify the description of the test.